media/css/protocol/components/_footer.scss

```diff
@@ -4,6 +4,11 @@
 
 @import '../../../protocol/css/includes/lib';
 
+.mzp-c-footer.sticky-overlay {
```

Comment:
Is `.sticky-overlay` an okay class name? Not sure if there's a better naming convention I can utilize.

Reply:
This is a little unusual since class names usually relate to the element itself and not some other element in the page. Maybe `intersect-sticky-overlay` is a little more descriptive?

Also suggest adding a comment in the CSS to explain why this exists.

Reply:
Comment added and class name updated.